composer.json

```
@@ -26,20 +26,21 @@
         "friends-of-behat/suite-settings-extension": "^1.0",
         "friends-of-behat/symfony-extension": "^2.1",
         "friends-of-behat/variadic-extension": "^1.3",
-        "lakion/mink-debug-extension": "^1.2.3",
-        "phpspec/phpspec": "^6.1",
-        "phpstan/phpstan": "0.12.25",
-        "phpstan/phpstan-doctrine": "0.12.13",
-        "phpstan/phpstan-webmozart-assert": "0.12.4",
-        "phpunit/phpunit": "^8.5",
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
+        "lakion/mink-debug-extension": "^2.0.0",
+        "phpspec/phpspec": "^7.0",
+        "phpstan/phpstan": "0.12.88",
+        "phpstan/phpstan-doctrine": "0.12.37",
+        "phpstan/phpstan-webmozart-assert": "0.12.12",
+        "phpunit/phpunit": "^9.5",
         "sensiolabs/security-checker": "^6.0",
         "sylius-labs/coding-standard": "^3.1",
-        "symfony/browser-kit": "^4.4",
-        "symfony/debug-bundle": "^4.4|^5.0",
-        "symfony/dotenv": "^4.4|^5.0",
-        "symfony/intl": "^4.4|^5.0",
-        "symfony/web-profiler-bundle": "^4.4|^5.0",
-        "symfony/web-server-bundle": "^4.4|^5.0"
+        "symfony/browser-kit": "^4.4 || ^5.2",
+        "symfony/debug-bundle": "^4.4|^5.0 || ^5.2",
+        "symfony/dotenv": "^4.4|^5.0 || ^5.2",
+        "symfony/intl": "^4.4|^5.0 || ^5.2",
+        "symfony/web-profiler-bundle": "^4.4|^5.0 || ^5.2",
+        "symfony/web-server-bundle": "^4.4|^5.0 || ^5.2"
```

```suggestion
        "symfony/debug-bundle": "^4.4 || ^5.2",
        "symfony/dotenv": "^4.4 || ^5.2",
        "symfony/intl": "^4.4 || ^5.2",
        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
        "vimeo/psalm": "4.6.4"
```
Please update `composer.json` according to the plugin skeleton

Make diff on both files.

for example you need to add
```
"config": {
        "sort-packages": true
    },
```
and remove 
```
    "prefer-stable": true,
```